services/blockchain-indexer/methods/dataService/events.js

```diff
@@ -24,6 +24,9 @@ module.exports = [
 			senderAddress: { optional: true, type: 'string' },
 			blockID: { optional: true, type: 'string' },
 			timestamp: { optional: true, type: 'string' },
+			topic: { optional: true, type: 'string' },
+			module: { optional: true, type: 'string' },
+			name: { optional: true, type: 'string' },
```

Comment:
Should we also add module and name to event indexes?
https://github.com/LiskHQ/lisk-service/blob/release/0.7.0/services/blockchain-indexer/shared/database/schema/events.js#L28

Reply:
There is no need to add indexes to the event table because we first fetch eventIDs based on query params from the `eventTopic` table and based on the eventIDs we query data from event table.